### PR TITLE
AWeber templates. Email Auth and Landing Pages

### PR DESCRIPTION
**Pull request by davestys**
Adding the first draft of templates for AWeber email authentication DNS records as well as Landing Page aliases.

### CODE REVIEW

File: aweber.com.landing-pages-subdomain.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+   "serviceId":"landing-pages-subdomain",
+   "serviceName":"AWeber Landing Pages",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure a domain to AWeber Landing Pages",
+   "variableDescription":"",
+   "records":[
+       {
+         "type": "A",
+         "host": "{%host%}",
```
Comment (pawel-kow) [inline aweber.com.landing-pages-subdomain.json:13]: See previous comment. `@` is the proper value.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: aweber.com.email-authentication-optin.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   "serviceName":"AWeber Email Authentication for Optin.com",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure AWeber's SPF and DKIM records to improve sender reputation and deliverability.",
+   "variableDescription":"{%subdomain%} is the AWeber customer's unique subdomain",
+   "records":[
+      {
+         "type": "SPFM",
+         "host": "%host%",
+         "spfRules": "v=spf1 include:{%account-subdomain%}.optin.com ~all"
```
[the remark this answers is not in the document]
Comment (davestys) [reply aweber.com.email-authentication-optin.json:14]: Understood. Is the "~" in "~all" guaranteed?

File: aweber.com.email-authentication-optin.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+   "serviceId":"email-authentication-optin",
+   "serviceName":"AWeber Email Authentication for Optin.com",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure AWeber's SPF and DKIM records to improve sender reputation and deliverability.",
+   "variableDescription":"{%subdomain%} is the AWeber customer's unique subdomain",
+   "records":[
+      {
+         "type": "SPFM",
+         "host": "%host%",
```
[the remark this answers is not in the document]
Comment (davestys) [reply aweber.com.email-authentication-optin.json:13]: Understood. So, will this work if we specify a subdomain for SPF? **Example**: mail.davestys.com

File: aweber.com.email-authentication.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
```
[the remark this answers is not in the document]
Comment (davestys) [reply aweber.com.email-authentication.json:1]: ok

File: aweber.com.landing-pages-subdomain.json (hunk 1 above)
Comment (davestys) [reply aweber.com.landing-pages-subdomain.json:13]: ok thank you for the feedback and guidance.

File: aweber.com.email-authentication-optin.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (pawel-kow) [reply aweber.com.email-authentication-optin.json]: 👍

File: aweber.com.email-authentication-optin.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (pawel-kow) [reply aweber.com.email-authentication-optin.json:14]: Yes, Please see https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#spf-record-merging-1

[the remark this answers is not in the document]
Comment (davestys) [reply aweber.com.email-authentication-optin.json]: thanks for the link. looks good.

File: aweber.com.email-authentication-optin.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+   "serviceName":"AWeber Email Authentication for Optin.com",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure AWeber's SPF and DKIM records to improve sender reputation and deliverability.",
+   "variableDescription":"{%subdomain%} is the AWeber customer's unique subdomain",
+   "records":[
+      {
+         "type": "SPFM",
+         "host": "@",
+         "spfRules": "include:{%account-subdomain%}.optin.com"
```
Comment (pawel-kow) [inline aweber.com.email-authentication-optin.json:14]: Please also remove curly brackets from the variable syntax. It's only `%variablename%`

Comment (davestys) [reply aweber.com.email-authentication-optin.json:14]: ty done.

**Review by davestys — COMMENTED** [review]
Fixed variable syntax for %account-domain%

**Review by pawel-kow — APPROVED** [review]
All good.

**Comment by pawel-kow** [comment]
@davestys good to merge? "First draft" in the title makes me unsure.

**Comment by davestys** [comment]
> @davestys good to merge? "First draft" in the title makes me unsure.

Ready to merge. Thank you @pawel-kow

**Pull request merged**